content/docs/practices/naming.md

```diff
@@ -6,24 +6,24 @@ sort_rank: 1
 # Metric and label naming
 
 The metric and label conventions presented in this document are not required
-for using Prometheus, but can serve as both a style-guide and collection of
+for using Prometheus, but can serve as both a style-guide and a collection of
 best practices. Individual organizations may want to approach e.g. naming
 conventions differently.
 
 ## Metric names
 
-A metric name:
+A metric name...
 
-* should have a (single-word) application prefix relevant to the containing Prometheus domain
- * <code><b>prometheus</b>\_notifications\_total</code>
- * <code><b>indexer</b>\_requests\_latencies\_milliseconds</code>
- * <code><b>processor</b>\_requests\_total</code>
-* must have a single unit (i.e. do not mix seconds with milliseconds)
-* should have a units suffix
+* ...should have a (single-word) application prefix relevant to the domain the metric belongs to. The prefix is sometimes referred to as `namespace` by client libraries. For metrics specific to an application, the prefix is usually the application name itself. Sometimes, however, metrics are more generic, like standardized metrics exported by client libraries. Examples:
+ * <code><b>prometheus</b>\_notifications\_total</code> (specific to the Prometheus server)
+ * <code><b>process</b>\_cpu\_seconds\_total</code> (exported by many client libraries)
+ * <code><b>http</b>\_request\_duration\_microseconds</code> (for all HTTP requests)
+* ...must have a single unit (i.e. do not mix seconds with milliseconds).
+* ...should have a suffix describing the unit.
```

Comment:
`_total` is not a really a unit. To make it even more confusing, we have metrics with just `_total` and others with `<unit>_total` (like `foobar_seconds_total`). Maybe add an extra comment that counters should have an additional `_total` suffix *after* the unit?